scss/detail.scss

```diff
@@ -7,11 +7,24 @@
 }
 
 .book-cover {
-  height: 100%;
-  margin-right: 52px;
-  max-height: 368px;
-  max-width: 261px;
-  width: 100%;
+  position: relative;
+
+  .cover {
+    height: 100%;
+    margin-right: 52px;
+    max-height: 368px;
+    max-width: 261px;
+    width: 100%;
+  }
+
+  .badge {
+    height: 100%;
+    max-height: 104px;
+    position: absolute;
+    right: 32px;
+    top: -16px;
```

Comment:
Try to use bottom instead, we avoid the use of negative numbers for absolute positions

Reply:
Fixed in PR #6